Model/CustomItemImportModel.php

```diff
@@ -94,22 +96,27 @@ public function import(Import $import, array $rowData, CustomObject $customObjec
             $merged = true;
         }
 
-        $this->linkContacts($customItem, $contactIds);
+        $this->linkContacts($customItem, $contactIds, $importLogDto);
 
         return $merged;
     }
 
     /**
      * @param int[] $contactIds
      */
-    private function linkContacts(CustomItem $customItem, array $contactIds): CustomItem
+    private function linkContacts(CustomItem $customItem, array $contactIds, ?ImportLogDTO $importLogDto): void
     {
         foreach ($contactIds as $contactId) {
+            if (!$this->entityManager->getRepository(Lead::class)->exists((string) $contactId)) {
+                $importLogDto?->addWarning(
```

Comment:
Please do not use any PHP 8 only construct in ACS until we release https://acquia.atlassian.net/browse/MAUT-11148. It might cause kafka consumer to fail.